addon/components/one-way-input.js

```diff
@@ -86,12 +86,16 @@ const OneWayInputComponent = Component.extend({
   },
 
   init() {
-    this._super(...arguments);
```

Comment:
`Ember.VERSION === '2.0.3'`

 ```js
init() {
  this._super(...arguments);
  let value = get(this, 'paramValue') || get(this, 'value');
  set(this, 'value', value);

  debugger;
  this._sanitizedValue = get(this, 'value') || get(this, 'checked');
},

didReceiveAttrs() {
  debugger;
  this._super(...arguments);
  this._processNewValue.call(this, undefined, get(this, get(this, 'appropriateAttr')));
}
```

The debugger in `didReceiveAttrs` gets hit first.